extensions/vertx-core/runtime/src/main/java/io/quarkus/vertx/core/runtime/config/AddressResolverConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.vertx.core.runtime.config;
+
+import java.util.OptionalInt;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class AddressResolverConfiguration {
+
+    /**
+     * The maximum amount of time in seconds that a successfully resolved address will be cached.
+     */
+    @ConfigItem
```

Comment:
I am thinking of wether to add a default value here, values can be taken from `AddressResolverOptions.DEFAULT_CACHE_MAX_TIME_TO_LIVE` This is mainly for the generated docs.

Reply:
you mean a default value within the ConfigItem annotation?

Reply:
Yes, exactly that e.g https://github.com/quarkusio/quarkus/blob/master/extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/DataSourceJdbcRuntimeConfig.java#L46

Reply:
Ok, I'll add default values as defined by AddressResolverOptions